content/influxdb/cloud/reference/release-notes/cloud-updates.md

```diff
@@ -14,10 +14,15 @@ aliases:
 InfluxDB Cloud updates occur frequently. Find a compilation of recent updates below.
 To find information about the latest Flux updates in InfluxDB Cloud, see [Flux release notes](/influxdb/cloud/reference/release-notes/flux/).
 
-<!-- ## January 2022
+## January 2022
+
 ### Update Tokens UI
 
-To enhance security, the Token UI will only display an InfluxDB Cloud token when it's first created. If you return to the Token page later, you will not be able to view or copy the token. -->
+To enhance security, the Tokens UI will only display an InfluxDB Cloud token when it's first created. If you return to the Token page later, you won't be able to view or copy the token.
```

Comment:
@jstirnaman, is there a good doc to link to for this change?

Reply:
@noramullen1 I think this would be a safe bet:
For more information, see how to [view tokens] (https://docs.influxdata.com/influxdb/cloud/security/tokens/view-tokens/).

Reply:
```suggestion
To enhance security, the Tokens UI will only display an InfluxDB Cloud token when it's [first created](https://docs.influxdata.com/influxdb/cloud/security/tokens/create-token/). If you return to the Token page later, you won't be able to view or copy the token. To learn more about token access restrictions, see [Create an API token](https://docs.influxdata.com/influxdb/cloud/security/tokens/create-token/)
```